### ISSUE TITLE
All newer opened files are automatically pinned

### ISSUE THREAD

**Issue by yfdyh000** [kylepaulsen/vscode-stack-tabs#6]
Issue Type: <b>Bug</b>

Enable this extension.
Click on a file to open, from vscode's file list (tree).
See the tab, it is quickly pinned.
Clicking another file will create a new pinned tab instead of using the tab.

Extension version: 0.0.2
VS Code version: Code 1.59.1 (3866c3553be8b268c8a7f8c0482c0c0177aa8bfa, 2021-08-19T11:56:46.957Z)
OS version: Windows_NT x64 10.0.19043
Restricted Mode: No

<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492:30256859
pythonvspyt639:30300192
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263cf:30335440
pythondataviewer:30285071
pythonvsuse255:30340121
vscod805cf:30301675
pythonvspyt200:30340761
vscextlang:30333561
binariesv615:30325510
pythonvssor306:30344512
bridge0708:30335490
vstre464cf:30350173
bridge0723:30353136
vsdyn478:30354110

```

</details>

**Comment by kylepaulsen** [comment]
I've known about this for a while, but I'm actually not sure what to do about it.
I've tried looking through vscode's api documentation, but I don't see any way to get how many pinned tabs there are (or to even check if a tab is pinned at all). You can find a thread about even getting a list of open editors here: https://github.com/microsoft/vscode/issues/15178 Even if such a thing existed, I'm not sure these openEditor objects would tell me if they are pinned or not.

Anyway, long story short, I'm not sure this is possible to fix right now. If anyone knows how to detect if a tab is pinned or count how many pinned tabs there are, please let me know.
